packages/bull-vault/test/integration-test/FlashBullTestFork.t.sol

```diff
@@ -153,6 +157,135 @@ contract FlashBullTestFork is Test {
         );
     }
 
+    function testSecondFlashDeposit() public {
+        uint256 ethToCrabInitial = 5e18;
+        (uint256 ethInCrab, uint256 squeethInCrab) = _getCrabVaultDetails();
+        (uint256 wSqueethToMint, uint256 fee) = _calcWsqueethToMintAndFee(
+            ethToCrabInitial,
+            squeethInCrab,
+            ethInCrab
+        );
+        uint256 crabToBeMinted = _calcSharesToMint(
+            ethToCrabInitial.sub(fee),
+            ethInCrab,
+            IERC20(crabV2).totalSupply()
+        );
+        uint256 bullCrabBalanceBefore = IERC20(crabV2).balanceOf(
+            address(bullStrategy)
+        );
+
+        uint256 bullShare = 1e18;
+        (uint256 wethToLend, uint256 usdcToBorrow) = bullStrategy
+            .calcLeverageEthUsdc(
+                crabToBeMinted,
+                bullShare,
+                ethInCrab,
+                squeethInCrab,
+                crabV2.totalSupply()
+            );
+
+        uint256 totalEthToBull = calcTotalEthToBull(
+            wethToLend,
+            ethToCrabInitial,
+            usdcToBorrow,
+            wSqueethToMint
+        );
+
+        vm.startPrank(user1);
+        flashBull.flashDeposit{value: totalEthToBull}(
+            ethToCrabInitial,
+            0,
+            0,
```

Comment:
I think we can set those to non-zero values, like getting the current prices from the pool and add a bit of slippage